src/templates/details.rs

```diff
@@ -102,9 +214,10 @@ async fn get_build_state(info: StateGeneratorInfo<()>) -> DetailState {
             external_id: "".to_owned(),
             description: "".to_owned(),
             academic_level: "".to_owned(),
-            optional_prerequisites: vec!["".to_owned()],
-            required_prerequisites: vec!["".to_owned()],
+            optional_prerequisites: vec!["null".to_owned()],
+            required_prerequisites: vec!["undefined".to_owned()],
```

Comment:
Any reason why these two are defined differently?

Reply:
i am poking fun at javascript, those are only displayed if the course does not exist